### DISCUSSION TITLE
vim.lsp.buf.signature_help multiple choice how to select?

### DISCUSSION THREAD

**Discussion by ssh352** [LazyVim/LazyVim#3055] [Q&A] [answered]
<img width="1046" alt="Screenshot 2024-04-25 at 15 37 08" src="https://github.com/LazyVim/LazyVim/assets/41779116/66432349-02bf-4101-aefa-1aae09176d9d">

with default Lazyvim configuration and lang.rust enabled, I press 'ctrl-k' for signature help. It presents two choices, How do I select 1 or 2? I tried tab, up down arrow, 1, 2 to no avail. Please help, Thanks!

**Comment by dpetka2001** [comment, marked as answer]
If you press `<C-k>` a second time, the cursor will go inside the lsp signature pop-up. Maybe then you can press `<Enter>` with your cursor on top of 1 or 2 to go to relevant section? Or if `<Enter>` doesn't work, maybe `K` or `<C-]>`? Not sure how rust-analyzer works.
